### ISSUE TITLE
Interested but doesn't seems to work

### ISSUE THREAD

**Issue by Felina-Lain** [Atvaark/ddda-save-editor#1]
I clicked the link, uploaded my save but then nothing happens, it ask my steam number and that's it...
shame, I would of liked an editor for this...

**Comment by Atvaark** [comment]
Hey,

sorry to disappoint you. I abandoned this project because my employer ended up not using this toolchain.

The only feature I wanted to add was changing the SteamID linked to the savegame so that you could share them. Would you mind uploading your savegame so I could see if the format was changed?

**Comment by Felina-Lain** [comment]
hey
ah well, back to text editor it is then XD

Sure though, how do I do that?



Marine Bocquet
sawula@free.fr 

 
From: Atvaark
Date: 2016-12-20 17:46
To: Atvaark/ddda-save-editor
CC: Felina; Author
Subject: Re: [Atvaark/ddda-save-editor] Interested but doesn't seems to work (#1)
Hey,
sorry to disappoint you. I abandoned this project because my employer ended up not using this toolchain.
The only feature I wanted to add was changing the SteamID linked to the savegame so that you could share them. Would you mind uploading your savegame so I could see if the format was changed?

**Comment by Atvaark** [comment]
Could you pack your DDDA.sav as a ZIP archive and attach it to your post here? Or upload it to mediafire.com and send me the link.

It doesn't contain anything personal besides your SteamID, which is public to all people that can view your Steam profile.

**Comment by Felina-Lain** [comment]
Sure
here you go :)



Marine Bocquet
sawula@free.fr 

 
From: Atvaark
Date: 2016-12-20 19:30
To: Atvaark/ddda-save-editor
CC: Felina; Author
Subject: Re: [Atvaark/ddda-save-editor] Interested but doesn't seems to work (#1)
Could you pack your DDDA.sav as a ZIP archive and attach it to your post here? Or upload it to mediafire.com and send me the link.
It doesn't contain anything personal besides your SteamID, which is public to all people that can view your Steam profile.

**Comment by vgchat** [comment]
This github needs to be closed then or updated for those of us looking for a working save editor.  Thanks.

**Comment by Atvaark** [comment]
I don't think you understand how this open source stuff works. You aren't in the position to demand anyone to to anything. Be it removing a project or making changes to it.
If a project doesn't do something you want **it's up to you to do it yourself**. You might as well pay someone professional to do it for if you actually want it done and you don't have the skills yourself.

**Comment by igormartimiano** [comment]
I mean it would be a lot nicer if you just have implied that it isn't working as intended on the readme, right? So people wouldn't lose their time testing it. Thanks anyway.
